packages/oasis-actions/src/actions/aave.ts

```diff
@@ -18,6 +18,7 @@ export function aaveDeposit(
       {
         asset: args.asset,
         amount: args.amount.toFixed(0),
+        setAsCollateral: args.setAsCollateral === undefined ? true : args.setAsCollateral,
```

Comment:
I'm thinking if this should be true as default.. This is our main use case at the moment, right?